pycaret/internal/pycaret_experiment/supervised_experiment.py

```diff
@@ -850,9 +850,14 @@ def compare_models(
                     [master_display, compare_models_], ignore_index=False
                 )
             master_display = master_display.round(round)
-            master_display = master_display.sort_values(
-                by=sort, ascending=sort_ascending
-            )
+            if self._ml_usecase != MLUsecase.TIME_SERIES:
+                master_display = master_display.sort_values(
+                    by=sort, ascending=sort_ascending
+                )
+            else:
+                master_display = master_display.sort_values(
+                    by=sort.upper(), ascending=sort_ascending
+                )
```

Comment:
What's the reason for separating it like this? Wouldn't upper work with both?

Reply:
Without converting to uppercase, it does not work for time_series since the argument is lower case but the metrics data frame is upper case. It works OK for regression/classification without this change.

Reply:
can we do the same for both?
```suggestion
            master_display = master_display.sort_values(
                by=sort.upper(), ascending=sort_ascending
            )
```